crates/ide-completion/src/context.rs

```diff
@@ -809,12 +826,76 @@ impl<'a> CompletionContext<'a> {
             qualifier_ctx,
             locals,
             depth_from_crate_root,
+            exclude_flyimport_traits,
+            exclude_traits,
             complete_semicolon,
         };
         Some((ctx, analysis))
     }
 }
 
+fn resolve_exclude_traits_list(db: &RootDatabase, traits: &[String]) -> FxHashSet<hir::Trait> {
+    let _g = tracing::debug_span!("resolve_exclude_trait_list", ?traits).entered();
+    let crate_graph = db.crate_graph();
+    let mut crate_name_to_def_map = FxHashMap::default();
+    let mut result = FxHashSet::default();
+    'process_traits: for trait_ in traits {
+        let mut segments = trait_.split("::").peekable();
+        let Some(crate_name) = segments.next() else {
+            tracing::error!("error resolving `{trait_}` from trait exclude lists: invalid path");
```

Comment:
nit:
```suggestion
            tracing::error!(?trait_, "error resolving trait from trait_exclude list: invalid path");
```

Reply:
There is no config named `trait_exclude`, so this might be confused. I tweaked this a bit.